code/modules/mob/living/carbon/xenomorph/castes/king/abilities_conqueror.dm

```diff
@@ -0,0 +1,979 @@
+	var/turf/current_turf = xeno_owner.loc
+	var/turf/new_turf = get_step_rand(living_target.loc)
+	for(var/x = 1 to get_dist(current_turf, new_turf))
+		var/turf_to_check = get_step(current_turf, get_dir(current_turf, new_turf))
+		if(turf_to_check && !DirBlocked(current_turf, turf_to_check, xeno_owner.pass_flags))
+			new /obj/effect/temp_visual/conqueror/dash_trail(turf_to_check, get_dir(current_turf, turf_to_check))
+		current_turf = turf_to_check
+	playsound(new_turf, 'sound/effects/alien/behemoth/landslide_enhanced_charge.ogg', 8, TRUE)
+	playsound(living_target, 'sound/effects/alien/conqueror/will_kick.ogg', 40, TRUE)
+	new /obj/effect/temp_visual/conqueror/hook/punch(living_target.loc)
+	xeno_owner.forceMove(new_turf)
+	living_target.do_jitter_animation(700, CONQUEROR_OBLITERATION_DEBUFF)
+	living_target.Immobilize(CONQUEROR_OBLITERATION_DEBUFF SECONDS)
+	living_target.adjust_stagger(CONQUEROR_OBLITERATION_DEBUFF)
+	living_target.adjust_slowdown(CONQUEROR_OBLITERATION_DEBUFF)
+	living_target.apply_damage((xeno_owner.xeno_caste.melee_damage * xeno_owner.xeno_melee_damage_modifier) * CONQUEROR_OBLITERATION_DAMAGE_MULTIPLIER, BRUTE, 0, MELEE, TRUE, TRUE, TRUE, xeno_owner.xeno_caste.melee_ap)
+	INVOKE_ASYNC(living_target, TYPE_PROC_REF(/mob, emote), "gored")
+	UnregisterSignal(living_target, list(COMSIG_QDELETING, COMSIG_MOB_DEATH, COMSIG_MOVABLE_Z_CHANGED))
+	targets_to_attack -= living_target
+	if(length(targets_to_attack))
+		attack_timer = addtimer(CALLBACK(src, PROC_REF(attack_targets)), CONQUEROR_OBLITERATION_ATTACK_DELAY, TIMER_UNIQUE|TIMER_OVERRIDE|TIMER_STOPPABLE)
+		return
+	disable_ability()
+
+/// Disables the ability.
+/datum/action/ability/xeno_action/conqueror_obliteration/proc/disable_ability()
+	SIGNAL_HANDLER
+	xeno_owner.playsound_local(xeno_owner.loc, 'sound/voice/hiss5.ogg', 25, TRUE)
+	xeno_owner.remove_movespeed_modifier(MOVESPEED_ID_CONQUEROR_OBLITERATION)
+	REMOVE_TRAIT(xeno_owner, TRAIT_NOPLASMAREGEN, CONQUEROR_OBLITERATION_TRAIT)
+	STOP_PROCESSING(SSprocessing, src)
+	ability_range = initial(ability_range)
+	xeno_owner.alpha = initial(xeno_owner.alpha)
+	xeno_owner.status_flags &= ~(GODMODE|INCORPOREAL)
+	UnregisterSignal(xeno_owner, COMSIG_MOB_DEATH)
+	if(toggled)
+		toggled = FALSE
+		set_toggle(FALSE)
+	if(particle_holder)
+		QDEL_NULL(particle_holder)
+	if(xeno_owner.client)
+		UnregisterSignal(xeno_owner.client, COMSIG_XENOABILITY_CONQUEROR_ENDURANCE_UP)
+	if(!xeno_owner.canmove)
+		xeno_owner.set_canmove(TRUE)
+	if(length(targets_to_attack))
+		for(var/mob/living/ref_living AS in targets_to_attack)
+			UnregisterSignal(ref_living, list(COMSIG_QDELETING, COMSIG_MOB_DEATH, COMSIG_MOVABLE_Z_CHANGED))
+			targets_to_attack -= ref_living
+	targets_to_attack = list()
+	if(attack_timer)
+		deltimer(attack_timer)
+
+/particles/obliteration_warning
+	icon = 'icons/effects/160x160.dmi'
+	icon_state = "obliteration_warning"
+	width = 225
+	height = 225
+	count = 1
+	spawning = 15
+	lifespan = 1
+	color = COLOR_VIOLET
```

Comment:
why are you using a particle for this instead of an effect

Reply:
Changed to an effect stored in the user's `vis_contents`. Will be in the next commit.